### PR TITLE
feat(i18n): add de

### PR DESCRIPTION
**Pull request by upupnoah**

### CODE REVIEW

File: locales/external/en.json [hunk 1]
```diff
@@ -1,31 +1,42 @@
 {
-  "copied_link": "Copied link to clipboard",
-  "feed.follower_one": "follower",
-  "feed.follower_other": "followers",
-  "feed.followsAndReads": "{{subscriptionCount}} {{subscriptionNoun}} with {{readCount}} {{readNoun}} on {{appName}}",
-  "feed.read_one": "read",
-  "feed.read_other": "reads",
-  "header.app": "App",
```
Comment (Innei) [inline locales/external/en.json:8]: Why change its structure?

**Review by Innei — CHANGES_REQUESTED** [review]

File: locales/external/en.json (hunk 1 above)
Comment (upupnoah) [reply locales/external/en.json:8]: > Why change its structure?

I'm a bit confused. Logically, the keys in a JSON file within a directory should be the same, so why are there some that start with copy_link and others that start with feed?

**Comment by Innei** [comment]
@upupnoah Could you take some time to solve the conflicts?

**Comment by upupnoah** [comment]
> @upupnoah Can you spare some time to resolve the conflicts?

Conflict has been resolved

**Comment by Innei** [comment]
@upupnoah CI's reporting an error, can you fix it?

**Comment by upupnoah** [comment]
> @upupnoah CI's reporting an error, can you fix it?

It has been fixed now.

File: apps/renderer/src/@types/constants.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // This file is auto-generated by scripts/generate-i18n-locale.ts
 // DONT EDIT THIS FILE MANUALLY
 export const currentSupportedLanguages = [
+  "de",
```
Comment (Innei) [inline apps/renderer/src/@types/constants.ts:4]: Moving down, `en` needs to be first

**Review by Innei — CHANGES_REQUESTED** [review]

File: apps/renderer/src/@types/constants.ts (hunk 2 above)
Comment (upupnoah) [reply apps/renderer/src/@types/constants.ts:4]: okey

**Review by Innei — APPROVED** [review]
Thank you

**Pull request merged**